src/App.tsx

```
@@ -27,17 +62,35 @@ function App() {
   const isProductionDomeWindowVisible = useGameStore((state) => state.isProductionDomeWindowVisible)
   const hideProductionDomeWindow = useGameStore((state) => state.hideProductionDomeWindow)
-  // --- ---
-  // --- NEW: Get Issue Window state and actions ---
-  const isIssueWindowVisible = useGameStore((state) => state.isIssueWindowVisible)
-  const hideIssueWindow = useGameStore((state) => state.hideIssueWindow)
-  const resolveIssue = useGameStore((state) => state.resolveIssue)
-  const getCurrentIssue = useGameStore((state) => state.getCurrentIssue)
-  const activeIssueId = useGameStore((state) => state.activeIssueId)
+  // --- Get Tutorial Window state and actions ---
+  const isTutorialWindowVisible = useGameStore((state) => state.isTutorialWindowVisible)
+  const hideTutorialWindow = useGameStore((state) => state.hideTutorialWindow)
```

ESLint

[error] 'hideTutorialWindow' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)